react/src/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" style="width: 100%; height: 100%">
+<head>
+  <title>React Blueprint for Foundry</title>
+
+  <link rel="stylesheet" href="../node_modules/@crowdstrike/tailwind-toucan-base/index.css" />
+  <link rel="stylesheet" href="../node_modules/@crowdstrike/falcon-shoelace/dist/style.css" />
```

Comment:
Question: will rollup understand these references and rewrite them, so that these files and up in `/dist` and not refer to `node_modules` anymore before uploading?

Reply:
Yes, that works well 👌🏻